### ISSUE TITLE
Accessibility: Add a link to skip navigation

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-admin#1968]
[WCAG2.2 2.4.1](https://www.w3.org/TR/WCAG22/#bypass-blocks)

We should make it easy for screen-reader and keyboard-navigation users to skip the left menu.

## Acceptance Criteria

- [x] The first focusable element in the CMS is a "skip navigation" link
- [x] The "skip navigation" link is invisible when it doesn't have focus
- [x] The "skip navigation" link moves focus to the main container on the page (usually the element with class `.cms-content`)
- [x] The changelog calls out any changes developers need to make to their template customisations (e.g. adding an id to a specific element) to make this functionality work.

## PRs
- https://github.com/silverstripe/silverstripe-admin/pull/1995
- https://github.com/silverstripe/developer-docs/pull/812

**Comment by GuySartorelli** [comment]
PRs merged
